### DISCUSSION TITLE
I was bored so I made a small website...

### DISCUSSION THREAD

**Discussion by loulou64490** [Jeffser/Alpaca#82] [Show and tell]
[https://alpaca-ai.pages.dev/](https://alpaca-ai.pages.dev/)

**Comment by Jeffser** [comment]
bruh I was going to do that for the 1.0.0 release, it looks great

**Comment by loulou64490** [comment]
I can modify it if you want, and give you all the source file, or host them on GitHub

**Comment by Jeffser** [comment]
I could host it on jeffser.com/alpaca

**Reply by loulou64490** [reply]
Yep

**Reply by loulou64490** [reply]
> I could host it on jeffser.com/alpaca

Here are the source files https://github.com/Jeffser/Alpaca/pull/85

**Reply by Jeffser** [reply]
Could you send me the files or maybe make a pull request to https://github.com/Jeffser/Jeffser.github.io
